lib/fizzy/execute.hpp

```diff
@@ -18,11 +18,14 @@
 
 namespace fizzy
 {
-// The result of an execution.
+/// The result of an execution.
 struct ExecutionResult
 {
+    /// This is true if the execution has trapped. No other fields are valid.
     const bool trapped = false;
+    /// This is true if the `value()` returns valid data.
```

Comment:
(there's no `value()`)
```suggestion
    /// This is true if value contains valid data.
```

Reply:
I think there may have been some rebases earlier.